spec/integration/_dev/benchmark/deploy/spec.yml

```diff
@@ -1,15 +0,0 @@
-spec:
```

Comment:
This file is still being used in the rally specification:
https://github.com/elastic/package-spec/blob/f2406bfb68971ed88c8e569cbc143c713b8bad6f/spec/integration/_dev/benchmark/spec.yml#L50

```
[2023-05-24T07:23:15.094Z]         	Error:      	Received unexpected error:
[2023-05-24T07:23:15.094Z]         	            	open integration/_dev/benchmark/deploy/spec.yml: file does not exist
[2023-05-24T07:23:15.094Z]         	            	could not open folder specification file
```

As this same specification is used for system and rally benchmarks, could it be recovered and just the same definition is used in both places?